.github/workflows/testing.yml

```diff
@@ -8,6 +8,7 @@ on:
   pull_request:
     branches:
       - main
+      - adding_timeseries
```

Comment:
This change is not necessary. It will test all merge requests into the main branch. Please remove.